src/app.js

```diff
@@ -1 +1,61 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+console.log('Hallo daar!');
+
+async function fetchData() {
```

Comment:
het lijkt mij, als leek, dat je zelf mag weten of je functies splitst of niet. Het maakt het vaak wel overzichtelijker. Ik zou dan wel harder moeten nadenken over waar ik al mijn errors 404, 500 afvang. Ik zou ook niet weten of je in regel 16 nogmaals een await moet plaatsen.